manifests/repo/redhat.pp

```diff
@@ -23,9 +23,9 @@
     priority   => 1,
   }
 
-  yumrepo { 'remi-php56':
```

Comment:
Might be better to cleanup if a version is changed by using a fixed-name repo and removing the old one, e.g.:

```puppet
yumrepo { 'remi-php56':
  ensure => absent,
}

yumrepo { 'remi-php':
  ...
}
```